cogs/utils/generic.py

```diff
@@ -65,4 +70,21 @@ async def get_channels(self, ctx):
     async def on_message_delete(self, message):
         pass
 
-# add blurbs, reminders, birthday alerts
\ No newline at end of file
+
+    # Send a message to a user on their birthday
+    @tasks.loop(hours=24)
+    async def birthday_alert(self):
+        members = await get_members_by_bday(date.today())
+        for member in members:
+            member_id = member['member_id']
+            user = self.bot.get_user(member_id)
```

Comment:
As as I was reviewing this, I just remembered that there are situations where the database might still hold data for a `member` but that member is no longer on any guild that the bot has access to. This would cause `.get_user()` to return `None`, which would cause an exception on `.send()` later down the line. A simple `if user:` check will avoid that exception.